docs/api/python/gluon/contrib.md

```diff
@@ -59,6 +59,17 @@ In the rest of this document, we list routines provided by the `gluon.contrib` p
     PixelShuffle3D
 ```
 
+### Convolutional neural network
+
+```eval_rst
+.. currentmodule:: mxnet.gluon.contrib.cnn
+
+.. autosummary::
+    :nosignatures:
+    
+    Deformable_Convolution
```

Comment:
Take a look at the website preview and you can see that this new function isn't linking up to the API details like the others.
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-14269/8/install/index.html?platform=Windows&language=R&processor=GPU

You are missing something like this down past line 120:
```
.. automodule:: mxnet.gluon.contrib.cnn
    :members:
    :imported-members:
```